### PR TITLE
HBK.TEDS-Editor version 1.5.0

### PR DESCRIPTION
**Pull request by lars-nvhgroup**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

### CODE REVIEW

**Comment by lars-whta** [comment]
> Hello @lars-whta,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

It becomes apparent when you look up/visit the domain in question: bksv.com.

**Comment by stephengillie** [comment]
Hi @lars-whta,

Could you add a PackageUrl?

(Automated response - build 843.)

**Comment by lars-whta** [comment]
> Hi @lars-whta,
> 
> Could you add a PackageUrl?
> 
> (Automated response - build 843.)

It does not have a typical package URL, only a download page:
https://www.bksv.com/fr/services/downloads/teds-editor-software/teds-editor-software-update
and a data sheet:
https://www.bksv.com/media/doc/Bp1856.pdf

I am not sure any of those would be correct to put into PackageUrl.

File: manifests/h/HBK/TEDS-Editor/1.5.0/HBK.TEDS-Editor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: HBK.TEDS-Editor
+PackageVersion: 1.5.0
+PackageLocale: en-US
+Publisher: Brüel & Kjær
+PackageName: Brüel & Kjær TEDS Editor BZ-5294
```
Comment (stephengillie) [inline manifests/h/HBK/TEDS-Editor/1.5.0/HBK.TEDS-Editor.locale.en-US.yaml:8]:
```suggestion
PackageName: Brüel & Kjær TEDS Editor BZ-5294
PacakgeUrl: https://www.bksv.com/fr/services/downloads/teds-editor-software/teds-editor-software-update
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
The download page works best here. We have a soft requirement that any user be able to navigate from the PackageUrl to the InstallerUrl, and this arrangement makes meeting that requirement very easy.

**Comment by lars-whta** [comment]
> Hello @lars-whta,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

See previous discussions above.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**